lagtraj/input_definitions/examples_files/forcings_conversion/dephy_inversion_nudging.yaml

```diff
@@ -0,0 +1,47 @@
+levels_method   : copy
```

Comment:
These 2 files are the same as the standard kpt and dephy conversion ones, only adding parameters for nudging above the inversion.